web/templates/page/index.html.eex

```diff
@@ -1,6 +1,6 @@
 <div id="game"></div>
 
 <div id="chat">
-    <input id="chat-input" type="text"></input>
+    <input id="chat-input" type="text" autofocus></input>
```

Comment:
makes it so you can just start typing as soon as the game is loaded without having to manually select the chat box